ibis-server/tests/routers/v2/connector/test_redshift.py

```diff
@@ -0,0 +1,352 @@
+        },
+    )
+    assert response.status_code == 422
+
+    response = await client.post(
+        url=f"{base_url}/validate/column_is_valid",
+        json={
+            "connectionInfo": connection_info,
+            "manifestStr": manifest_str,
+            "parameters": {"modelName": "Orders", "columnName": "X"},
+        },
+    )
+    assert response.status_code == 422
+
+
+async def test_validate_rule_column_is_valid_without_parameters(client, manifest_str):
+    response = await client.post(
+        url=f"{base_url}/validate/column_is_valid",
+        json={"connectionInfo": connection_info, "manifestStr": manifest_str},
+    )
+    assert response.status_code == 422
+    result = response.json()
+    assert result["detail"][0] is not None
+    assert result["detail"][0]["type"] == "missing"
+    assert result["detail"][0]["loc"] == ["body", "parameters"]
+    assert result["detail"][0]["msg"] == "Field required"
+
+
+async def test_validate_rule_column_is_valid_without_one_parameter(
+    client, manifest_str
+):
+    response = await client.post(
+        url=f"{base_url}/validate/column_is_valid",
+        json={
+            "connectionInfo": connection_info,
+            "manifestStr": manifest_str,
+            "parameters": {"modelName": "Orders"},
+        },
+    )
+    assert response.status_code == 422
+    assert response.text == "Missing required parameter: `columnName`"
+
+    response = await client.post(
+        url=f"{base_url}/validate/column_is_valid",
+        json={
+            "connectionInfo": connection_info,
+            "manifestStr": manifest_str,
+            "parameters": {"columnName": "orderkey"},
+        },
+    )
+    assert response.status_code == 422
+    assert response.text == "Missing required parameter: `modelName`"
+
+
+async def test_metadata_list_tables(client):
+    response = await client.post(
+        url=f"{base_url}/metadata/tables",
+        json={"connectionInfo": connection_info},
+    )
+    assert response.status_code == 200
+    tables = response.json()
+    assert len(tables) >= 1
+
+    test_table = next(filter(lambda t: "orders" in t["name"].lower(), tables), None)
+    if test_table:
+        assert test_table["name"] is not None
+        assert test_table["properties"] is not None
+        assert test_table["properties"]["schema"] == "public"
+        assert len(test_table["columns"]) > 0
+
+        # Check column structure
+        column = test_table["columns"][0]
+        assert column["name"] is not None
+        assert column["type"] is not None
+        assert "notNull" in column
+
+
+async def test_metadata_list_constraints(client):
+    response = await client.post(
+        url=f"{base_url}/metadata/constraints",
+        json={"connectionInfo": connection_info},
+    )
+    assert response.status_code == 200
+
+    result = response.json()
+    # fixme Redshift does support constraints
+    assert len(result) == 0
```

Comment:
```suggestion
    # TODO: Redshift does support constraints
    assert len(result) == 0
```
Is it an issue? If Redshift doesn't support constraints, I think we don't need to note it `fixme` or `todo`.